tests/multigrid-global-coarsening/events_non_nested.output

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
@fdrmrc  It looks like this test depends on trilinos. Could you rename the file!? I'll merge once the tests have passed!

Reply:
So done, sorry for that